index.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>mobile version portfolio</title>
+    <script src="https://kit.fontawesome.com/e008fa04f4.js" crossorigin="anonymous"></script>
+    <link href="style.css" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+      <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+</head>
+<body>
+   <header class="main-header">
+    <div class="header-container">   
+    <a href="" class="logo text-color">Solomon</a>
+    <nav>
+        <span class="material-symbols-outlined text-color">
+            menu
+            </span>
+    </nav>
+    </div> 
+   </header>
+   <section class="section-1">
+       <div class="heading-1-container">
+            <h1 class="heading-1 margin-24">Hi there,</h1>
+            <h1 class="heading-1 margin-24">Solomon</h1>
+       </div>
+       <h2 class="heading-2 text-color margin-24">I'm a software Developer</h2>
+       <p class="section-par text-color margin-24">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+    see and have a project you need coded,
+    don’t hestiate to contact me.</p>
+    
+        <div class="image-0">
+          <img src="socialmedia/design.png" alt="image">
+        </div>
+
+   </section>
+
+
+   <section class="section-2">
+     <div class="gridd">
+      <div class="my-recent-works">
+       <div class="H1">
+         <h1 > MY RECENT WORK</h1>
+       </div>
+      <div class="line">
+      <img  src="image/line.png" alt="line">
+      </div>
+      <div class="main-image">
+        <img src="image/image1.png" alt="main image">
+      </div>
+      <div class="H2">
+        <h2 > Multi-post Stories</h2>
+      </div>
+       <div class="p2">
+        <p>A daily selection of privately personalized reads;
+         no accounts or sign-ups required. has been the industry's standard
+         dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+       </div>
+       <div>
+      <ul class="buttons-0">
+        <li> <button class="btn" type="button">CSS</button></li>
+        <li> <button class="btn" type="button">HTML</button></li>
+        <li> <button  class="btn" type="button">Bootstrap</button></li>
+        <li> <button class="btn" type="button">Ruby</button></li>
+      </ul>
+      </div>
+      <div class="see-project">
+        <a class="seep" href="#"> see project</a>
+      </div>
+      </div>
```

Comment:
- [ ] Kindly remove the `margin-left: 20px;` because it's not exist on the Figma design.
- [ ] Kindly make the space between the header and the image smaller to match the Figma design.
- [ ] Kindly add to each project technology some padding to match the Figma design.
- [ ] Kindly center the text inside the `see project` button and make the text white to match the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/166127597-3dff3a03-f491-420e-bbec-eb13de77d117.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166127732-16449ad3-309e-49a0-b301-07095e44ae33.png)